src/utils.ts

```diff
@@ -39,6 +39,20 @@ import logger from './logger';
 
 dotenv.config();
 
+export const IGNORE_TXS = {
```

Comment:
I don't like having these hashes here. Maybe move to the constants file?

Reply:
Do we have the wallet-lib installed here? I think we should have the genesis hashes there and use it in the wallets. We already have this in the wallet-service, explorer and daemon

Reply:
We don't, but I agree that would be a good idea.

I'll merge and open an issue to refactor it when it's ready